dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -236,6 +244,26 @@ def execute(self, sql: str, *parameters: Any) -> None:
         self._cde_connection.delete_resource(job_name)
         logger.debug("{}: Done delete resource".format(job_name))
 
+    """
+    Fetch spark driver stderr log and create a new log file with job name which has failed
+    """
+    def log_spark_driver_errors(self, job, job_name):
+        try:
+            events, failed_job_stderr = self._cde_connection.get_job_output(
+                job_name, job, log_type="stderr"
+            )
+
+            error_file_name = job_name + ".stderr.log"
+            directory = os.path.join(os.getcwd(), "logs", error_file_name)
```

Comment:
since log paths may be configured, you probably need the config object: config.log_path

Reply:
Resolving as log-path parameter is not currently being recognised.